content/05.conclusion.md

```diff
@@ -1,18 +1,26 @@
-The majority of research involving bioRxiv focuses on the metadata of preprints; however, the language contained within these preprints has not previously been systematically examined.
+Over 77% of bioRxiv preprints that were published ended up within Pubmed Central's Open Access Corpus (PMCOA) which is a significant improvement compared to the earlier years [@doi:10.7287/peerj.preprints.3520v1].
+This highlights a beneficial shift towards more preprint authors publishing their work in an open access format.
+The majority of research involving bioRxiv focuses on the metadata of preprints; however, the language contained within these preprints has yet to be systematically examined.
 Throughout this work, we sought to analyze the language within these preprints and understand how it changes in response to peer review.
+
 Our global corpora analysis found that writing within bioRxiv is consistent with the biomedical literature in the PMCOA repository, suggesting that bioRxiv is linguistically similar to PMCOA.
 Token-level analyses between bioRxiv and PMCOA suggested that research fields drive significant differences; e.g., more patient-related research is prevalent in PMCOA than bioRxiv.
 This observation is expected as preprints focused on medicine are supported by the complementary medRxiv repository [@url:https://www.medrxiv.org/].
 Token-level analyses for preprints and their corresponding published version suggest that peer review may focus on data availability and incorporating extra sections for published papers; however, future studies are needed to ascertain individual token level changes as preprints venture through the publication process.
+Furthermore, it would be interesting to perform a token comparison between  preprints and accepted author manuscripts within Pubmed Central as this may highlight non-journal related differences. 
 
 Document embeddings are a versatile way to examine language contained within preprints, understanding peer review's effect on preprints, and provide extra functionality for preprint repositories.
+We used a linear approach to generate document embeddings as our main goal was to increase model interpretability; however, future work could look into incorporating more advanced models to generate document vectors such as Doc2Vec[@arxiv:arXiv:1405.4053] or BERT [@doi:10.1093/bioinformatics/btz682].
 Examining linguistic variance within document embeddings of life science preprints revealed that the largest source of variability was informatics.
 This observation bisects the majority of life science research categories that have integrated preprints within their publication workflow.
+Along with anaylyzing lingusitic variation, future work could utilize this embedding space to quantify sentiment trends such as authors writing preprints with a more positive language over time.
+Furthermore, future applications similar to [@doi:10.1038/s41586-019-1335-8] could be applied to this embedding space.
```

Comment:
What is the application? Someone shouldn't have to read the entire reference to understand the sentence :)